angel_system/berkeley/detectron2/engine/train_loop.py

```diff
@@ -258,10 +258,10 @@ def __init__(self, model, data_loader, optimizer):
         new = []
         num_para = 0
         temp = self.optimizer.param_groups[0]
-        for paras in temp['params']:
-            num_para += 1
-            if num_para < 115:
-                paras.requires_grad = False
+        #for paras in temp['params']:
+        #    num_para += 1
+        #    if num_para < 115:
+        #        paras.requires_grad = True
```

Comment:
Should this code be deleted?

Reply:
Probably not? I was told by berkeley to remove it if I was trying to fully retrain a model. But if we were fine-tuning then this would still apply

Reply:
OK. I'm sure this could be parameterized but not now.